pkg/controllermanager/apis/config/v1alpha1/defaults.go

```diff
@@ -147,3 +147,13 @@ func SetDefaults_LeaderElectionConfiguration(obj *LeaderElectionConfiguration) {
 		obj.LockObjectName = ControllerManagerDefaultLockObjectName
 	}
 }
+
+// SetDefaults_EventControllerConfiguration sets defaults for the EventControllerConfiguration.
+func SetDefaults_EventControllerConfiguration(obj *EventControllerConfiguration) {
```

Comment:
Can you add a small unit test? https://github.com/gardener/gardener/blob/master/pkg/controllermanager/apis/config/v1alpha1/defaults_test.go

Reply:
added